lib/terrafying/components/dynamicset.rb

```diff
@@ -33,7 +33,7 @@ def create_in(vpc, name, options = {})
         options = {
           public: false,
           eip: false,
-          ami: aws.ami('base-image-fc-2860fb52', owners = ['136393635417']),
+          ami: aws.ami('base-image-fc-3c48f829', owners = ['136393635417']),
```

Comment:
the owner should be for the artefacts account. `477284023816`

Reply:
ah right :+1:

Reply:
updated :+1: